src/modules/navigator/rtl.cpp

```diff
@@ -423,9 +434,8 @@ void RTL::set_rtl_item()
 				_mission_item.yaw = _destination.yaw;
 			}
 
-			if (_navigator->get_vstatus()->is_vtol
-			    || _navigator->get_vstatus()->vehicle_type == vehicle_status_s::VEHICLE_TYPE_FIXED_WING) {
-				_mission_item.loiter_radius = _rtl_loiter_rad;
+			if (_navigator->get_vstatus()->vehicle_type == vehicle_status_s::VEHICLE_TYPE_FIXED_WING) {
+				_mission_item.loiter_radius = landing_loiter_radius;
```

Comment:
should we explicitly set the loiter radius here even if it is not a fixed-wing? it is not set before the switch case, and in this case the waypoint type is a loiter. is it already defaulted somewhere else?

Reply:
You're right, let's just always fill all the fields, independent on vehicle type.
https://github.com/PX4/PX4-Autopilot/pull/20214/commits/b9ddb4510936d5da3d10b662b9f130ec63b0a7a5